.changeset/forty-meals-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-core": minor
```

Comment:
Should there be a line here for `perseus` since there are changes there as well?

Reply:
Good catch! I guess this is one reason not to generate the changeset until the PR is reviewed 🤔